src/pyhf/contrib/viz/brazil.py

```diff
@@ -1,13 +1,314 @@
+         observed :math:`\mathrm{CL}_{s+b}` line.
+
+        clb (:class:`matplotlib.lines.Line2D`): The artist of the optional
+         observed :math:`\mathrm{CL}_{b}` line.
     """
+
+    def __init__(
+        self,
+        cls_obs,
+        cls_exp,
+        one_sigma_band,
+        two_sigma_band,
+        test_size,
+        clsb=None,
+        clb=None,
+        **kwargs,
+    ):
+        r"""
+        Args:
+            cls_obs (:class:`matplotlib.lines.Line2D`): The observed
+             :math:`\mathrm{CL}_{s}` line.
+            cls_exp (:obj:`list` of :class:`matplotlib.lines.Line2D`): The
+             expected :math:`\mathrm{CL}_{s}` lines.
+            one_sigma_band (:class:`matplotlib.collections.PolyCollection`): The
+             :math:`\mathrm{CL}_{s,\mathrm{exp}}` :math:`\pm1\sigma` bands.
+            two_sigma_band (:class:`matplotlib.collections.PolyCollection`): The
+             :math:`\mathrm{CL}_{s,\mathrm{exp}}` :math:`\pm2\sigma` bands.
+            test_size (:class:`matplotlib.lines.Line2D`): The line indicating the
+             test size, :math:`\alpha`.
+            clsb (:class:`matplotlib.lines.Line2D`): The optional observed
+             :math:`\mathrm{CL}_{s+b}` line.
+            clb (:class:`matplotlib.lines.Line2D`): The optional observed
+             :math:`\mathrm{CL}_{b}` line.
+        """
+        super().__init__(**kwargs)
+        self.cls_obs = cls_obs
+        self.cls_exp = cls_exp
+        self.one_sigma_band = one_sigma_band
+        self.two_sigma_band = two_sigma_band
+        self.test_size = test_size
+        self.clsb = clsb
+        self.clb = clb
+
+    def get_children(self):
+        """
+        Return a list of the child Artists of this Artist.
+        """
+        # Unpack everything so that self.draw can work
+        artists = [
+            self.cls_obs,
+            *self.cls_exp,
+            self.one_sigma_band,
+            self.two_sigma_band,
+            self.test_size,
+            self.clsb,
+            self.clb,
+        ]
+        return tuple(artist for artist in artists if artist is not None)
+
+    def draw(self, renderer, *args, **kwargs):
+        """
+        Draw the Artist (and its children) using the given renderer.
+
+        This has no effect if the artist is not visible (``.Artist.get_visible``
+        returns ``False``).
+
+        .. note::
+
+            Even though ``draw`` is in the public API of
+            :class:`matplotlib.artist.Artist` users should probably not call
+            it themselves unless they are familiar with ``matplotlib``'s
+            renderer backend mechanics.
+
+        Args:
+            renderer: :class:`matplotlib.backend_bases.RendererBase` subclass.
+        """
+        if not self.get_visible():
+            return
+
+        for artist in self.get_children():
+            artist.draw(renderer, *args, **kwargs)
+        self.stale = False
```

Comment:
what's the point of stale?

Reply:
Used by `matplotlib` for some state bookkeeping for artists:

> Whether the artist is 'stale' and needs to be re-drawn for the output to match the internal state of the artist.

https://matplotlib.org/stable/api/_as_gen/matplotlib.artist.Artist.stale.html